scene/resources/texture.cpp

```diff
@@ -38,23 +38,61 @@
 #include "scene/resources/bit_map.h"
 #include "servers/camera/camera_feed.h"
 
+int Texture2D::get_width() const {
+	int ret;
+	if (GDVIRTUAL_CALL(_get_width, ret)) {
+		return ret;
+	}
+	ERR_FAIL_V_MSG(0, "Unimplemented method");
```

Comment:
You used "function" in all other classes, would be good to do in Texture2D too for consistency.